src/vaishnaviVJun20_arrayExamples/Fibonacci.java

```diff
@@ -0,0 +1,24 @@
+package vaishnaviVJun20_arrayExamples;
+
+//To print first n Fibonacci Numbers
+
+public class Fibonacci {
+	
+	void printFibonacciNumbers(int input) {
```

Comment:
The current logic is giving **11** elements instead of **10**. Check output screenshot.
**1 1  2  3  5  8  13  21  34  55  89**

Reply:
-- In requirement it is mentioned that we need to print first and fibonnaci numbers.
--So according to given output as well I think we do need to print 11 numbers.